ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -148,8 +148,6 @@ <h2 class="heading-2">{{ _("Contact details") }}</h2>
 {% block scripts %}
     {{ super() }}
     <script async src="{{ mathjax() }}"></script>
-    <script async src="{{ static('js/third-party/pym.min.js') }}"></script>
```

Comment:
We can't go ahead with this change I'm afraid, as it's a security risk. Let's just try moving it to the head, so that it's more likely to be loaded by the time the other script runs.

Reply:
This has now been reverted.